### ISSUE TITLE
Dancing Rune Weapon (DRW) do not copy my abilities

### ISSUE THREAD

**Issue by KreepAK** [Duskhaven/Bug-Reports#1566]
DRW - Summons a second rune weapon that fights on its own for 12 sec, doing the same attacks as the Death Knight but for 50% reduced damage and healing.

DRW is summoned as a debuff on the target, dealing some dmg (I can stay away and watch) absolutely not related to the dmg I am doing to the target. 
What it suppose to do is - appear as additional weapon (model is the same as the weapon is equipped when DRW is used) near by and attack the target that I am attacking with the same abilities that I am using but with dmg reduction to 50% from the dmg I am doing. 

In other words, if I use Heart Strike and crit for 2k, DRW should deal 1k.

**Comment by KreepAK** [comment]
Any updates regarding this issue?
